docs/reference/licenses.md

```diff
@@ -1,36 +1,36 @@
 ---
 id: licenses
 title: "Licenses"
-description: "Licensing information for components of Camunda Platform 8"
+description: "Licensing information for components of Camunda 8"
 ---
 
 ## Licensing
 
-This page contains licensing information for all components of Camunda Platform 8.
+This page contains licensing information for all components of Camunda 8.
 
 ### Zeebe
 
 Licenses and license information for Zeebe can be found in the Zeebe project [README](https://github.com/camunda/zeebe#license).
 
 ### Operate, Tasklist, Identity, Optimize
 
-Operate, Tasklist, Identity, and Optimize are licensed to our users and customers under a proprietary license. The proprietary license can be either the [Camunda Platform Self-Managed Free Edition license](https://legal.camunda.com/licensing-and-other-legal-terms#legal-terms-camunda-platform-c8-self-managed-free-edition-and-beta-offerings) or the Camunda Platform Self-Managed Enterprise Edition license (a copy you obtain when you contact Camunda). Note that while the Camunda Platform Self-Managed Free Edition is indeed free, it only allows usage of the software in non-production environments. To use the software in production, [purchase the Camunda Platform Self-Managed Enterprise Edition](https://camunda.com/platform/camunda-platform-enterprise-contact/). This licensing information also applies to web apps and APIs.
+Operate, Tasklist, Identity, and Optimize are licensed to our users and customers under a proprietary license. The proprietary license can be either the [Camunda Self-Managed Free Edition license](https://legal.camunda.com/licensing-and-other-legal-terms#legal-terms-camunda-platform-c8-self-managed-free-edition-and-beta-offerings) or the Camunda Self-Managed Enterprise Edition license (a copy you obtain when you contact Camunda). Note that while the Camunda Self-Managed Free Edition is indeed free, it only allows usage of the software in non-production environments. To use the software in production, [purchase the Camunda Self-Managed Enterprise Edition](https://camunda.com/platform/camunda-platform-enterprise-contact/). This licensing information also applies to web apps and APIs.
 
 ### Connectors
 
 The Connector SDK, REST Connector, Connector Runtime Docker image, and Connectors Bundle Docker image are licensed under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0). All other Connectors provided by Camunda out of the box are licensed to our users and customers under the same proprietary license as the one defined for Operate, Tasklist, Identity, and Optimize above.
 
 ### Web Modeler
 
-Web Modeler is licensed under the proprietary Camunda Platform Self-Managed Enterprise Edition license (a copy you obtain when you contact Camunda).
+Web Modeler is licensed under the proprietary Camunda Self-Managed Enterprise Edition license (a copy you obtain when you contact Camunda).
```

Comment:
Can you confirm the the Web Modeler and/or legal team that this license name should change?

Reply:
Adjusted in commit 👍